GSA_Adapter/Enums/Output_Axis.cs

```diff
@@ -20,29 +20,20 @@
  * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
  */
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using BH.oM.Structure.Loads;
-using Interop.gsa_8_7;
-
-namespace BH.Engine.GSA
+namespace BH.Adapter.GSA
 {
-    public static partial class Query
-    {
-        /***************************************************/
-        /**** Public Methods                            ****/
-        /***************************************************/
 
-        public static string TaskType(LoadCombination comb)
-        {
-            if (comb.CustomData.ContainsKey("Task Type"))
-                return comb.CustomData["Task Type"].ToString();
+    /***************************************/
 
-            return "STATIC";
-        }
+    //In lack of an "Estring" function we have created a static class which returns
+    //the three different options of outputaxis. It's used when extracting info in SIGSA.
+    public static class Output_Axis
+    {
+        static public string Default() { return "default"; }
+        static public string Global() { return "global"; }
+        static public string Local() { return "local"; }
     }
```

Comment:
I can't understand why this has to be a class with methods returning static strings.
The fact that this file is included in the `enum` folder makes the case for this to become an actual enum, in case, or just a collection of static strings?

Reply:
This is to some extent a kind of old lefter over from before, so could see us solving this in some other manner. Would you prefer moving these to the adapter itself or to the convert class?

Reply:
Depends how you solve it - happy either way if it's fit